lms/djangoapps/course_structure_api/v0/serializers.py

```diff
@@ -10,6 +11,7 @@ class CourseSerializer(serializers.Serializer):
     """ Serializer for Courses """
     id = serializers.CharField()  # pylint: disable=invalid-name
     name = serializers.CharField(source='display_name')
+    subject = serializers.SerializerMethodField()
```

Comment:
This possibly can be replaced with `ChoiceField` http://www.django-rest-framework.org/api-guide/fields/#choicefield